api/src/client/ApiTaxClearanceCertificateClient.ts

```diff
@@ -0,0 +1,71 @@
+import { TaxClearanceCertificateResponse } from "@businessnjgovnavigator/shared";
+import { TaxClearanceCertificateClient } from "@domain/types";
+import { LogWriterType } from "@libs/logWriter";
+import { LookupTaxClearanceCertificateAgenciesById } from "@shared/taxClearanceCertificate";
+import { UserData } from "@shared/userData";
+import axios from "axios";
+
+type Config = {
+  orgUrl: string;
+  userName: string;
+  password: string;
+};
+
+export const ApiTaxClearanceCertificateClient = (
+  logWriter: LogWriterType,
+  config: Config
+): TaxClearanceCertificateClient => {
+  const postTaxClearanceCertificate = async (
+    userData: UserData
+  ): Promise<TaxClearanceCertificateResponse> => {
+    const logId = logWriter.GetId();
+    logWriter.LogInfo(`Tax Clearance Certificate Client - Id:${logId}`);
+
+    const currTaxClearanceData = userData.businesses[userData.currentBusinessId].taxClearanceCertificateData;
+
+    const postBody = {
+      repId: userData.user.id,
+      repName: userData.user.name,
+      taxpayerId: currTaxClearanceData.taxId,
+      taxpayerPin: currTaxClearanceData.taxPin,
+      taxpayerName: currTaxClearanceData.businessName,
+      addressLine1: currTaxClearanceData.addressLine1,
+      addressLine2: currTaxClearanceData.addressLine2,
+      city: currTaxClearanceData.addressCity,
+      state: currTaxClearanceData.addressState?.shortCode,
+      zip: currTaxClearanceData.addressZipCode,
+      agencyName: LookupTaxClearanceCertificateAgenciesById(currTaxClearanceData.requestingAgencyId).name,
+    };
+
+    logWriter.LogInfo(
+      `Tax Clearance Certificate Client - Id:${logId} - Request Sent to ${
+        config.orgUrl
+      }//TYTR_ACE_App/ProcessCertificate/businessClearance data: ${JSON.stringify(postBody)}`
+    );
+
+    try {
+      const response = await axios.post(
```

Comment:
The docs reference an endpoint with 2 `//` before TYTR_ACE_App, is this correct? Should we use one? It works in bruno with 1 and 2.
`${config.orgUrl}//TYTR_ACE_App`

Reply:
use 1 /